python/artm/regularizers.py

```diff
@@ -232,7 +232,7 @@ def __init__(self, name, tau, gamma, config, topic_names, class_ids, dictionary)
             try:
                 if config.HasField('dictionary_name'):
                     self._dictionary_name = config.dictionary_name
-            except:
+            except ValueError:
```

Comment:
I suggest to rewrite such try-except clause, because `HasField` method works not as someone can expect: quote from [StackOverflow](https://stackoverflow.com/a/36742793)

"I think you've misunderstood what HasField() does. It does not check whether a protobuf type defines a specific field by a name. What it does is to check if, given the name of a message field, that that field is set for the current instance."

My suggestion:
`
try:
    dict_name = config.dictionary_name
    self._dictionary_name = config.dictionary_name
except AttributeError:
    pass
`